src/Core/src/Handlers/DatePicker/DatePickerHandler.Android.cs

```diff
@@ -19,9 +19,6 @@ protected override MauiDatePicker CreatePlatformView()
 
 			var date = VirtualView?.Date;
 
-			if (date != null)
-				_dialog = CreateDatePickerDialog(date.Value.Year, date.Value.Month, date.Value.Day);
```

Comment:
Removing these lines would probably break a lot of apps, where they use a workaround like [this one](https://github.com/dotnet/maui/issues/12899#issuecomment-1403661769) for the datepicker unfocus event.

Reply:
Hmmm, thanks for pointing it out. I will test performance with these lines. 
Also I think the most problematic was DateTime either way so it might be alright to leave it.